website/pages/docs/from-1.5/blocks/packer.mdx

```diff
@@ -0,0 +1,100 @@
+---
+layout: docs
+page_title: packer - Blocks
+sidebar_title: <tt>packer</tt>
+description: |-
+  The "packer" configuration section is used to configure some behaviors
+  of Packer itself.
+---
+
+# Packer Settings
+
+`@include 'from-1.5/beta-hcl2-note.mdx'`
+
+The special `packer` configuration block type is used to configure some
+behaviors of Packer itself, such as requiring a minimum Packer version to
+apply your configuration.
+
+## Packer Block Syntax
+
+Packer settings are gathered together into `packer` blocks:
+
+```hcl
+packer {
+  # ...
+}
+```
+
+Each `packer` block can contain a number of settings related to Packer's
+behavior. Within a `packer` block, only constant values can be used;
+arguments may not refer to named objects such as resources, input variables,
+etc, and may not use any of the Packer language built-in functions.
+
+The various options supported within a `packer` block are described in the
+following sections.
+
+## Specifying a Required Packer Version
+
+The `required_version` setting accepts a [version constraint
+string,](#version-constraints) which specifies which versions of Packer
+can be used with your configuration.
+
+If the running version of Packer doesn't match the constraints specified,
+Packer will produce an error and exit without taking any further actions.
+
+Use Packer version constraints in a collaborative environment to
+ensure that everyone is using a specific Packer version, or using at least
+a minimum Packer version that has behavior expected by the configuration.
+
+
+## Version Constraints
+
+Anywhere that Packer lets you specify a range of acceptable versions for
+something, it expects a specially formatted string known as a version
+constraint.
+
+### Version Constraint Syntax
+
+Packer's syntax for version constraints is very similar to the syntax used by
+other dependency management systems like Bundler and NPM.
+
+```hcl
+version = ">= 1.2.0, < 2.0.0"
```

Comment:
the option is `required_version`, not version -- this got me very confused, because when I supplied the Packer block with just "version", the build ran successfully without complaint even though the constraint wasn't met and the field was the wrong name. Can we set our version finder to make sure we strictly look for the right key on the Packer block?

Reply:
Oops ! 🤦🏼  Yes, missed it after a rename.